bundles/org.openhab.binding.roborock/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -228,6 +229,7 @@
 	<channel-type id="msg-seq" advanced="true">
```

Comment:
Why is this relevant to the user? Even if it is advanced, it seems rather technicall and only for developers?